NEWgIT.Client/Pages/Index.razor

```diff
@@ -254,4 +259,10 @@
         public DateOnly Date { get; set; }
         public int Commits { get; set; }
     }
+
+    public record AuthorsDTO(Dictionary<string, Dictionary<DateOnly, int>> AuthorFrequencies);
+    
+    public record ForksDTO(ICollection<string> Forks);
+
+    public record FrequenciesDTO(Dictionary<DateOnly, int> Frequencies);

```

Comment:
for at undgå dependency på Newgit.Core - vi må lige overveje hvad der er det "rigtige" at gøre